app_bikes/settings.py

```diff
@@ -0,0 +1,3 @@
+VALUES_UPDATE_INTERVAL = 10
+STAR_API_KEY = ''
+VELIB_API_KEY = ''
```

Comment:
Why have those here if they're empty? Did you mean to have the base urls in here instead of in the provider modules?